composer.json

```diff
@@ -46,7 +46,8 @@
     "autoload": {
         "psr-4": {
             "Yiisoft\\Rbac\\Php\\": "src"
-        }
+        },
+        "files": ["tests/bootstrap.php"]
```

Comment:
There is no tests folder in the final vendor, remove this please

Reply:
Sorry, missed this one. Fixed in #81.